eks-distro-base/scripts/clean_docs

```diff
@@ -16,15 +16,14 @@
 
 set -e
 set -o pipefail
-set -x
```

Comment:
I think we can make these scripts a bit less noisy now, esp since we are tracking the full file list now...